api/specs.proto

```diff
@@ -122,6 +123,17 @@ message NetworkAttachmentSpec {
 	string container_id = 1 [(gogoproto.customname) = "ContainerID"];
 }
 
+// PluginSpec specifies runtime parameters for a plugin.
+message PluginSpec {
+	// image defines the image reference, as specified in the
+	// distribution/reference package. This may include a registry host, name,
+	// tag or digest.
+	string image = 1;
+
+	// enabled defines if the plugin should be enabled or not.
+	bool enabled = 2;
```

Comment:
`docker plugin install` defaults to enable. and it's also `Enabled` in `docker/docker/api/types/plugin.go`